docs/sphinx/source/installation.rst

```diff
@@ -19,9 +25,21 @@ If you are on a PC you should also copy the `Jaloxa radwinexe-5.0.a.8-win64.zip
 
```

Comment:
By "if you are in a PC", do you mean "if you use Windows"? If so, this should be corrected. Also, if you are NOT using Windows, is there anything else to do? What are these executables for, anyway? Please, clarify these.

Reply:
Added 

If you are on a Windows computer you should also copy the `Jaloxa radwinexe-5.0.a.8-win64.zip  <http://www.jaloxa.eu/resources/radiance/radwinexe.shtml>`_ executables into ``program files/radiance/bin``. This executables allow for some nifty visualization options of your generated scene inside of bifacial_radiance, like falsecolor images.